stdlib/2/SimpleHTTPServer.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for SimpleHTTPServer (Python 2)
+
+from typing import Any, AnyStr, IO, Mapping, Optional, Union
+import BaseHTTPServer
+from StringIO import StringIO
+
+class SimpleHTTPRequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    server_version = ...  # type: str
+    def do_GET(self) -> None: ...
+    def do_HEAD(self) -> None: ...
+    def send_head(self) -> Optional[IO[str]]: ...
+    def list_directory(self, path: AnyStr) -> Optional[StringIO]: ...
```

Comment:
I kind of agree. I'm not sure AnyStr should have been added to typing, since it's frequently misunderstood and it's really much more common that you want `Union[str, unicode]`.